lib/log_tools/exporter.rb

```diff
@@ -150,11 +150,20 @@ def export(*logfiles)
 
                     exporter = exporter_class.new(stream)
                     index = 0
+                    start_index_local = 0
+                    if start_index
+                        start_index_local = start_index
+                    end
+                    index_increment_local = 1
+                    if index_increment
+                        index_increment_local = index_increment
+                    end
                     time = Time.now
                     Exporter.info " exporting stream #{stream.name} (#{stream.size} samples): "
                     stream.samples.each do |rt,lg,sample|
```

Comment:
There is a `raw_each` that does not apply a conversion to Ruby (which is un-done at line 174). Would you mind fixing that ?